### ISSUE TITLE
2.x what is the recommended way to use the archaius1 bridge ?

### ISSUE THREAD

**Issue by brharrington** [Netflix/archaius#305]
I've been trying to use the archaius1 bridge, but it doesn't work if the static block of ConfigurationManager gets executed before the configure method of StaticArchaiusBridgeModule. That is pretty fragile in practice and typically it means archaius1 is getting mapped to the default implementation. 

If I set the system properties being set by the module on the java command I get an NPE because the static injection hasn't occurred yet:

```
Caused by: java.lang.RuntimeException: Error initializing configuration
	at com.netflix.config.ConfigurationManager.<clinit>(ConfigurationManager.java:109)
	... 28 more
Caused by: java.lang.NullPointerException
	at com.netflix.archaius.bridge.StaticDeploymentContext.getValue(StaticDeploymentContext.java:71)
	at com.netflix.config.ConfigurationManager.setDeploymentContext(ConfigurationManager.java:351)
	at com.netflix.config.ConfigurationManager.<clinit>(ConfigurationManager.java:96)
	... 28 more
```

**Comment by elandau** [comment]
I noticed this as well.  I need to modify the 0.x code to allow this.  Currently, once the static block has been called it's impossible to change.

**Comment by elandau** [comment]
I could modify the bridge to not set the archaius 0.x system properties but instead call,
```java
ConfigurationManager.install(this);
```
from the constructor of the bridge.

**Comment by elandau** [comment]
Fixed in #306
